.github/workflows/gradle-wrapper.yml

```diff
@@ -1,18 +0,0 @@
-name: gradle-wrapper
-
-on:
-  pull_request:
-    paths:
-      - 'gradlew'
-      - 'gradlew.bat'
-      - 'gradle/wrapper/'
-
-jobs:
-  validate:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Validate Gradle Wrapper
-        uses: gradle/wrapper-validation-action@v3
```

Comment:
Note: originally (eb34f8a1) this was extracted to a separate workflow because it was annoyingly timing out on network requests. This won't happen now because of changes in [2.1.0](https://github.com/gradle/wrapper-validation-action/releases/tag/v2.1.0).

The new `gradle/actions/setup-gradle` embeds the code of the wrapper-validation-action, which means we can get stable validation "for free" on every single build.